docker/Dockerfile.amd

```diff
@@ -0,0 +1,53 @@
+##############################################################
+# This Dockerfile contains AMD compilers
+##############################################################
+FROM rocm/dev-ubuntu-20.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+ARG rocm=/opt/rocm
+
+# Some utils needed
+RUN apt-get update && apt-get install -y wget git autoconf dh-autoreconf flex
+
+# roc-aware mpi. Taken from:
```

Comment:
how stable this is is a mistery 😬 

and won't we need to have something akin to docker-nvidia available on the runner?

Reply:
No amd runs with standard docker

Reply:
"ROCm-aware MPI. Taken from:"